overview.js

```diff
@@ -63,7 +63,23 @@ async function onOverviewLeaderboard (message, state) {
   await message.send('```\n' + table.toString() + '\n```')
 }
 
+async function onHelp (message, state) {
+  const helpText = `Here's all the fitness commands:
+* \`#std\`: records stairs session, stands for "stairs done" (${state.config.floors} by default)
+* \`#std <floors>\`: records stairs session of specific amount of floors (e.g. \`#std 5\`)
+* \`#stairs-lead\`: shows stairs leaderboard
+* \`#stairs-achievements\`: shows stairs achievements
+* \`#btw\`: records bike session, stands for "biked to work"
+* \`#btw <distance>\`: records bike session of specific distance (e.g. \`#btw 6.2\`), the first time you do it will set as default
+* \`#btw <distance> #save\`: records bike session and saves over previous default
+* \`#bike-lead\`: shows bike leaderboard
+* \`#bike-achievements\`: shows bike achievements
+* \`#lead\`: shows company leaderboard
+* \`#help\`: show this message`
+  message.send(helpText)
```

Comment:
Not sure if these were async or not. For mine I added `await` before `message` but not sure if needed. @valeriangalliat  ?